tests/unit/test_browse.py

```diff
@@ -444,16 +462,27 @@ def test_convert_3_multiband_to_raster(self):
         assert_array_equal(expected_raster, actual_raster.data)
 
     def test_convert_4_multiband_to_raster(self):
+        """Input data has NaN _fillValue match in the red layer at [1,1]
+        and alpha chanel also exists with a single transparent value at [0,0]
```

Comment:
chanel?

Reply:
lol, that reminds me of playing some guessing game and the answer was "Chanel #5", but the person was trying to tell us it was above nbc but below cbs. 💀